bundlewrap/repo.py

```diff
@@ -181,6 +181,8 @@ def __getattr__(self, attrname):
 
 class Repository(MetadataGenerator):
     def __init__(self, repo_path=None):
+        super().__init__()
+
```

Comment:
But is certainly more correct this way.

Should also move
```python
        # required by MetadataGenerator
        self._node_metadata_proxies = {}
        self._node_metadata_lock = RLock()
```
into `MetadataGenerator.__init__` then.

Reply:
Will fix tomorrow.